quizzes/models.py

```diff
@@ -0,0 +1,148 @@
+import uuid
+
+from django.contrib.postgres.fields import JSONField
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Question(models.Model):
+    QUESTION_TYPE_CHOICES = (
+        ('multiplechoice', "Multiple Choice"),
+        ('truefalse', "True/False"),
+        ('freetext', "Freeform Input")
+    )
+    """
+    Base Question model
+    """
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    version = models.IntegerField(default=1)
+    question_type = models.CharField(max_length=50,
+                                     choices=QUESTION_TYPE_CHOICES)
+    question = models.CharField(max_length=200)
+    answers = JSONField(null=True, blank=True)
```

Comment:
At some point I think it would be useful to validate that the answers are appropriate for the question type - for instance the `freetext` should not allow an array.